### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by mohanned-png**
My code has no comment -̈

### CODE REVIEW

File: src/country.py [hunk 1]
```diff
@@ -25,16 +25,16 @@ def __init__(self, name: str, iso3: str) -> None:
         """
         self.name = name
         self.iso3 = iso3
-        #TODO
-
+        Country.name_to_countries[name]=self
+        self.cities=list()
```
Comment (Ariges770) [inline src/country.py:29]: In general best practice is to have a space between the operator signs for example here `self.cities = list()`

File: src/country.py [hunk 2]
```diff
@@ -75,7 +95,12 @@ def add_city_to_country(city: City, country_name: str, country_iso3: str) -> Non
     :param country_iso3: The unique 3-letter identifier of this country
     :return: None
     """
-    #TODO
+    if Country.name_to_countries.get(country_name) == None:
```
Comment (Ariges770) [inline src/country.py:98]: I believe here `if country_name in Country.name_to_countries:` should work as you are checking if the country_name key is in the dict

**Review by Ariges770 — APPROVED** [review]
Have a look at the comments before merging but otherwise it looks good (also if you merge, don't delete the branch

**Review by mohanned-png — COMMENTED** [review]
None

**Review by Ariges770 — COMMENTED** [review]
You used `"Cities {}".format(self.cities)` or something like that which I don't believe is best practices

**Review by Ariges770 — APPROVED** [review]

**Review by Ariges770 — APPROVED** [review]

**Review by mohanned-png — COMMENTED** [review]
cool stuff here

**Pull request merged**